### DISCUSSION TITLE
“过滤抽奖号”是什么意思？

### DISCUSSION THREAD

**Discussion by L1cardo** [LeoChen98/BiliRaffle#3606] [Q&A] [answered]
![image](https://github.com/LeoChen98/BiliRaffle/assets/33802186/06bc5b6e-a83a-4737-a174-195776371a35)

**Comment by LeoChen98** [comment, marked as answer]
最新10条动态中抽奖动态含量满足阈值数量即判定为抽奖号被过滤
